step_implementations/specs/admin_pipeline_page_spec.rb

```diff
@@ -205,6 +205,46 @@
   admin_pipeline_page.add_user_to_group(user,type,group)
 end
 
- step 'Adding <user> as a <type> role for group <group>' do |user,type,group|
+step 'Adding <user> as a <type> role for group <group>' do |user,type,group|
   admin_pipeline_page.add_role_to_group(user,type,group)
 end
+
+step 'Verify pipeline group <group> has user <user> with <permissions> permissions' do |group,user,permissions|
+  users=admin_pipeline_page.users_in_group()
+  permissions=admin_pipeline_page.users_permissions_in_group(user)
+  assert_true users.include?user
+  permissions.each{|permission|
+  assert_true permissions.include?permission
```

Comment:
Here the asset is on `permissions` which is a list, if the purpose is to just check if given permission is in the list, why to loop on all elements of the list?